soroban-sdk/src/env.rs

```diff
@@ -428,8 +449,7 @@ impl Env {
     /// Replaces the executable of the current contract with the provided Wasm.
     ///
     /// The Wasm blob identified by the `wasm_hash` has to be already present
-    /// on-chain (the upload happens via `INSTALL_CONTRACT_CODE` host function
-    /// or via `install_contract_wasm` test function in unit tests).
+    /// on-chain (uploaded via `upload_contract_wasm` host function).
```

Comment:
The developer doesn't interact with host functions, they interact with SDK functions. We should refer to them instead.
```suggestion
    /// on-chain (uploaded via `[Deployer::upload_contract_wasm]` host function).
```